docs/cookbook/New-NSX-V-WLD.md

```diff
@@ -0,0 +1,272 @@
+# New NSX-V Workload Domain
+
+## DESCRIPTION
+In this example you will create an NSX-V backed VCF workload domain with the API.
+
+Once the initial VCF bringup is completed you need to add a workload domain(s) to service your workloads.
+
+The sequence of events is as follows:
+
+- Load the PowerVCF Module
+- Connect to SDDC Manager
+- Create a network pool
+- Commission hosts
+- Create Workload domain
+
+### Load the PowerVCF Module
+- Open powershell and cd to the directory where you downloaded the PowerVCF module
+- Run the following to import the module
+-     Import-Module .\PowerVCF
+
+### Connect to SDDC Manager
+To establish a session with SDDC Manager run the following
+
+```powershell
+Connect-VCFManager -fqdn sddc-manager.sfo01.rainpole.local -username admin -password VMw@re1!
+```
+### Create a network pool
+
+The first thing you need before you can commission new hosts is to create a new network pool, which will include the vSAN & vMotion network details for this workload domain cluster.
+
+Before you can create a network pool you first need to create the json body that will be passed in.
+
+#### TIP: The PowerVCF Module includes a folder of sample json files to get you started
+
+Here is the json format required for creating a vSAN network pool (Please use the same json with the module rather than copying from here as formatting is probably messed up!)
+
+```json
+
+{
+"name": "sfo01w01-cl01",
+"networks": [
+{
+"type": "VSAN",
+"vlanId": 2240,
+"mtu": 9000,
+"subnet": "172.16.240.0",
+"mask": "255.255.255.0",
+"gateway": "172.16.240.253",
+"ipPools": [
+{
+"start": "172.16.240.5",
+"end": "172.16.240.100"
+}
+]
+},
+{
+"type": "VMOTION",
+"vlanId": 2236,
+"mtu": 9000,
+"subnet": "172.16.236.0",
+"mask": "255.255.255.0",
+"gateway": "172.16.236.253",
+"ipPools": [
+{
+"start": "172.16.236.5",
+"end": "172.16.236.100"
+}
+]
+}
+]
+}
+
+```
+
+So first off lets get a list of current Network Pools. To do this run the following cmdlet:
+
+```powershell
+Get-VCFNetworkPool
+```
+
+As expected this returns a single network pool.
+
+#### TIP
+You can manipulate the return data in several ways.
+
+To create a new network pool using the json we created earlier run the following:
+
+New-VCFNetworkPool -json .\SampleJSON\NetworkPool\addNetworkPoolSpec.json
+
+Now running Get-VCFNetworkPool should display 2 Network Pools
+
+### Commission Hosts
+
+Now that you have a network pool you can commission hosts and associate them with the network pool. For this you need the following json
+
+#### TIP 
+For this json you need the network pool name & ID. These were returned when the pool was created and also by Get-VCFNetworkPool
```

Comment:
Change 'and also by Get-VCFNetworkPool' to 'and can also be obtained using Get-VCFNetworkPool'

Reply:
Done